dask/dataframe/core.py

```diff
@@ -4433,18 +4433,21 @@ def explode(self, column):
         meta = self._meta.explode(column)
         return self.map_partitions(M.explode, column, meta=meta, enforce_metadata=False)
 
-    def to_bag(self, index=False):
+    def to_bag(self, index=False, format="tuple"):
         """Convert to a dask Bag of tuples of each row.
 
         Parameters
         ----------
         index : bool, optional
             If True, the index is included as the first element of each tuple.
             Default is False.
+        format : tuple or dict,optional  default:tuple
+            returns bag of tuple or dict, based on this format
+            parameter.
```

Comment:
Thank you for updating the docstring here. Dask uses `numpydoc` for docstring formatting (xref https://numpydoc.readthedocs.io/en/latest/format.html). For this particular case, I think it should look something like:

```suggestion
        format : {"tuple", "dict"}
            Whether to return a bag of tuples or dictionaries.
```

Reply:
Thanks, for letting me know about `numpydoc`, this is new to me !! and  implemented accordingly